### ISSUE TITLE
Speaker overlap between tr and cv

### ISSUE THREAD

**Issue by srdfjy** [facebookresearch/svoice#50]
Hi，I look at the data set used in "https://github.com/enk100/speaker_separation/raw/master/txt_files_multi_speakers.zip", the training set and the verification set will overlap with speakers, that is, the speakers in the training set will appear In the validation set, will this affect the generalization performance of the model?

tr：si_tr_s
cv：si_tr_s
tt：si_dt_05

**Comment by adiyoss** [comment]
Hi @srdfjy,
You are right, there is an overlap between the speakers in the train and dev, and this might affect generalization performance. Ideally, we would like to have a completely different set of speakers. However, since have limited amount of speakers (~110) and would like to: i) mix 4 or 5 different speakers simultaneously speaking; ii) generate a (relatively) large amount of training examples, we decided to allow this overlap. 
Since there is not performance gap between the dev and test sets, we found that there is not generalization issue due to the speakers identity. 
In case you do observe issues related to the speakers identity, you generate a mixtures dataset using Librispeech, which has much more speakers than WSJ (~2200).
